app/graphql/types/input/recipes/update_recipe_type.rb

```diff
@@ -0,0 +1,17 @@
+module Types
+  module Input
+    module Recipes
+      class UpdateRecipeType < Types::BaseInputObject
+        argument :id, Integer, required: true
+        argument :user_id, Integer, required: false
+        argument :image, String, required: false
+        argument :title, String, required: false
+        argument :description, String, required: false
+        argument :instructions, String, required: false
+        argument :charity_id, String, required: false
+        argument :charity_name, String, required: false
+        argument :ingredients, [Types::Input::Recipes::RecipeIngredientType], required: false
```

Comment:
Just to clarify, FE can't update ingredients during updateRecipe? They'd have to update ingredients separately?

Reply:
Exactly, updateRecipeIngredients would be a different mutation. As I was building this updateRecipe mutation, I was trying to include ingredients as recipe attributes that can be updated. When I talked to @hashmaster3k about it though, I came away with the understanding that it may be better to separate the mutations. Certainly something we could refactor down the line as well